h2o-algos/src/main/java/hex/kmeans/KMeans.java

```diff
@@ -438,6 +439,15 @@ public void computeImpl() {
         DKV.remove(bestOutputKey);
       }
     }
+    
+    @Override
+    public void computeEffectiveParameters() {
```

Comment:
any reason this cant be in model class?

Reply:
originally I was calling model.initEffectiveParam() in the constructor so _output was not available yet and cases where output was needed I was solving separately. how the solution evolved I missed that I have it available at the moment when I call model.initEffectiveParam now. So all this can be now moved to model class. that's also the reason why it is called init... but now it can be called compute... will fix

Reply:
fixed